src/main/java/cz/inventi/kpj/spring/springintroduction/config/GreetingProperties.java

```diff
@@ -0,0 +1,18 @@
+package cz.inventi.kpj.spring.springintroduction.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
It will work but the @EnableConfigurationProperties is a better solution. You should add this property to any configuration class.

Try to work with lombok next time.  A lot of project is using that. It will be very useful to be familiar with that.